.github/workflows/main.yml

```diff
@@ -0,0 +1,146 @@
+name: OpenDingux_buildroot
+
+on: [push, pull_request]
```

Comment:
Forgot to mention, but I've configured the builds so they run on every push and every PR (when a PR is created, a build will be created and the result will appear on the PR itself). This affects every branch, but can also be configured on a branch basis if needed, plus some other trigger types (like a manual one, but I don't think it's useful in this case)

Reply:
So:
* For pushes, only for `opendingux` branch
* For PRs, for every PR? or for PRs targetting the `opendingux` branch as well?

Reply:
For PRs targetting the `opendingux` branch, I don't think we'll get PRs that target anything else anyway.